pkg/pgmodel/migrations/sql/1_base_schema.up.sql

```diff
@@ -1127,6 +1127,14 @@ AS $func$
 $func$
 LANGUAGE SQL STABLE PARALLEL SAFE;
 
+DO $$
```

Comment:
I think we want to separate out stub-of-functions in extension and extension-install into a separate migrate script just for clarity. can we do that?

Reply:
I not sure we should, we'll end up four different scripts, with weird dependencies: